pkg/apis/bundle/v1alpha1/patch_template_types.go

```diff
@@ -52,6 +52,9 @@ const (
 	// Merge Patch to apply patches. See more at:
 	// https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md
 	StrategicMergePatch PatchType = "StrategicMergePatch"
+
+	// Use JSON patch for non-kubernetes types, such as custom resources.
```

Comment:
I would change tho comment to be something like "JSONPatch relies on RFC6902 JSON Patching to apply patches and is useful for when objects are not rendered in the patcher runtime schema."

Reply:
done